lib/robots/dor_repo/accession/provenance_metadata.rb

```diff
@@ -9,34 +9,32 @@ def initialize
         end
 
         def perform(druid)
-          obj = Dor.find(druid)
-          build_datastream(obj, 'accessionWF', 'DOR Common Accessioning completed')
+          workflow_provenance = create_workflow_provenance(druid)
+          object_client = Dor::Services::Client.object(druid)
+          object_client.metadata.legacy_update(
+            provenance: {
+              updated: Time.now,
```

Comment:
I'm not sure it matters, but do we care if the timestamps in the `updated` field here and the WF XML event node differ?

Reply:
Not important because, this is only relevant to dor-services-app when it sees if it's newer than what is stored.